src/menu/views/file_info.c

```diff
@@ -285,6 +309,14 @@ static void draw (menu_t *menu, surface_t *d) {
         uint8_t memory_type = rom_db_match_expansion_pak(temp_header);
         sprintf(str_buffer,"Expansion PAK: %s\n", get_rom_memorytype_s(memory_type));
         graphics_draw_text(d, x_start_position, y_position += font_vertical_pixels, str_buffer);
+        y_position += (font_vertical_pixels * 2);
+
+        sprite_t *boxart_sprite = boxart_sprite_load(temp_header.metadata.unique_identifier);
```

Comment:
This is bad because it loads boxart every frame then unloads it after drawing. This could break when this screen will be rewritten to use rdpq. Loading data should be done once in `view_file_info_init` function.
Same exact thing happens in line 287. ROM header is loaded every frame again and again.

Reply:
Fixed.